src/menu.js

```diff
@@ -1,77 +1,71 @@
-import {activeButton} from './page'
-
 const menuItems = [
-    {
-       name: "Bambara",
-       desc: "factory functions simply set up and return the new object when you call the function"
-    },
-    {
-        name: "Champaer",
-        desc: " factory functions simply set up and return the new object when you call the function"
-     },
-     {
-        name: "Nocora",
-        desc: "factory functions simply set up and return the new object when you call the function"
-     },
-     {
-        name: "Lamaria",
-        desc: "factory functions simply set up and return the new object when you call the function"
-     },
-     
-
-]
+  {
+    name: 'Bambara',
+    desc: 'factory functions simply set up and return the new object when you call the function',
+  },
+  {
+    name: 'Champaer',
+    desc: ' factory functions simply set up and return the new object when you call the function',
+  },
+  {
+    name: 'Nocora',
+    desc: 'factory functions simply set up and return the new object when you call the function',
+  },
+  {
+    name: 'Lamaria',
+    desc: 'factory functions simply set up and return the new object when you call the function',
+  },
 
-function menuLoader(){
-    let menuButton = document.createElement('button')
-    menuButton.textContent = 'Menu'
-    menuButton.classList.add('menu_btns')
-    menuButton.classList.add('menuClick')
+];
 
-    menuButton.addEventListener("click", () => {
-        menuButton.classList.add("activeBtn")
-   
-        if(document.body.contains(document.querySelector('.bodyStyle'))){
-            document.querySelector('.bodyStyle').remove()
-        }
+function menuContent() {
```

Comment:
Required

As mentioned by the last reviewer, you should use ES6 features extensively. Kindly refactor some functions in your `menu.js` and `page.js` files to arrow functions.